### PR TITLE
Update to ISSUE-132

### PR DESCRIPTION
**Pull request by lewismc**
Hi Folks, this pull request has morphed into a very general code cleanup covering more or less every package.
The reason this pull request ended up this way is that I am still unable to ingest logs locally. This has meant that I've spent a hellish amount of time debugging a variety of data structures present across the MUDROD workflow. Whenever I go to look at and debug a new class I end up making code improvements and cleaning things up.
This pull request can therefore be considered as an enhancement to the codebase but not an addition of any particular functionality. 
If possible, I would like to have a Webex and discuss it in detail as soon as possible @fgreg @Yongyao @quintinali 
Thanks

### CODE REVIEW

File: core/src/main/java/gov/nasa/jpl/mudrod/ssearch/ClickstreamImporter.java [hunk 1]
```diff
@@ -48,12 +50,12 @@ public ClickstreamImporter(Properties props, ESDriver es, SparkDriver spark) {
...
     XContentBuilder Mapping;
     try {
-      Mapping = jsonBuilder().startObject().startObject(props.getProperty("clickstreamMatrixType")).startObject("properties").startObject("query").field("type", "string")
+      Mapping = jsonBuilder().startObject().startObject(props.getProperty(MudrodConstants.CLICK_STREAM_MATRIX_TYPE)).startObject("properties").startObject("query").field("type", "string")
           .field("index", "not_analyzed").endObject().startObject("dataID").field("type", "string").field("index", "not_analyzed").endObject()
 
           .endObject().endObject().endObject();
 
-      es.getClient().admin().indices().preparePutMapping(props.getProperty("indexName")).setType(props.getProperty("clickstreamMatrixType")).setSource(Mapping).execute().actionGet();
+      es.getClient().admin().indices().preparePutMapping(props.getProperty("indexName")).setType(props.getProperty(MudrodConstants.CLICK_STREAM_MATRIX_TYPE)).setSource(Mapping).execute().actionGet();
```
Comment (quintinali) [inline core/src/main/java/gov/nasa/jpl/mudrod/ssearch/ClickstreamImporter.java]: please change props.getProperty("indexName")  to props.getProperty(MudrodConstants.INDEX_NAME) in this line

File: core/src/main/java/gov/nasa/jpl/mudrod/ssearch/ClickstreamImporter.java [hunk 2]
```diff
@@ -63,7 +65,7 @@ public void addClickStreamMapping() {
    * Method to import click stream CSV into Elasticsearch
    */
   public void importfromCSVtoES() {
-    es.deleteType(props.getProperty("indexName"), props.getProperty("clickstreamMatrixType"));
+    es.deleteType(props.getProperty("indexName"), props.getProperty(MudrodConstants.CLICK_STREAM_MATRIX_TYPE));
```
Comment (quintinali) [inline core/src/main/java/gov/nasa/jpl/mudrod/ssearch/ClickstreamImporter.java]: please change props.getProperty("indexName") to props.getProperty(MudrodConstants.INDEX_NAME) in this line

File: core/src/main/java/gov/nasa/jpl/mudrod/ssearch/ClickstreamImporter.java [hunk 3]
```diff
@@ -78,7 +80,7 @@ public void importfromCSVtoES() {
         String[] clicks = line.split(cvsSplitBy);
         for (int i = 1; i < clicks.length; i++) {
           if (!clicks[i].equals("0.0")) {
-            IndexRequest ir = new IndexRequest(props.getProperty("indexName"), props.getProperty("clickstreamMatrixType"))
+            IndexRequest ir = new IndexRequest(props.getProperty("indexName"), props.getProperty(MudrodConstants.CLICK_STREAM_MATRIX_TYPE))
```
Comment (quintinali) [inline core/src/main/java/gov/nasa/jpl/mudrod/ssearch/ClickstreamImporter.java]: please change props.getProperty("indexName") to props.getProperty(MudrodConstants.INDEX_NAME) in this line

File: core/src/main/java/gov/nasa/jpl/mudrod/ssearch/ClickstreamImporter.java (hunk 1 above)
Comment (lewismc) [reply core/src/main/java/gov/nasa/jpl/mudrod/ssearch/ClickstreamImporter.java]: Done, thanks @quintinali

File: core/src/main/java/gov/nasa/jpl/mudrod/ssearch/ClickstreamImporter.java (hunk 2 above)
Comment (lewismc) [reply core/src/main/java/gov/nasa/jpl/mudrod/ssearch/ClickstreamImporter.java]: Done, thanks @quintinali

File: core/src/main/java/gov/nasa/jpl/mudrod/ssearch/ClickstreamImporter.java (hunk 3 above)
Comment (lewismc) [reply core/src/main/java/gov/nasa/jpl/mudrod/ssearch/ClickstreamImporter.java]: Done, thanks @quintinali

**Pull request merged**